docs/about/release-notes/migration-faq.md

```diff
@@ -0,0 +1,237 @@
+```
+
+```{dropdown} How do I manage different dependencies for separate stages, especially if packaging as Docker images?
+
+Each stage can specify its Conda environment, which must be present in the Docker image. Import your dependencies within stage logic to ensure proper isolation.
+```
+
+---
+
+## Deployment, Infrastructure, and Practicalities
+
+```{dropdown} How do I deploy NeMo Curator or Ray clusters?
+
+Ray clusters can be deployed on any major cloud platform (AWS, GCP, Azure) using standard Ray tools (see [Ray documentation](https://docs.ray.io/en/latest/cluster/getting-started.html)). No custom infrastructure is needed. NVIDIA provides ready-to-use {ref}`Docker images <reference-infrastructure-container-environments>` and up-to-date quickstart guides. For complete deployment details, refer to {ref}`Production Deployment Requirements <admin-deployment-requirements>`.
+```
+
+```{dropdown} Do I need to build custom Docker images?
+
+Not for most standard uses. Use {ref}`provided images <reference-infrastructure-container-environments>` or extend as needed (for example, to add proprietary or additional filters). Check out the official Docker container releases on the [NGC Catalog](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo-curator).
+```
+
+```{dropdown} Is there integration with existing cluster or orchestration tools like Airflow or Slurm?
+
+Ray manages its own orchestration natively. For more traditional batch job orchestration, Airflow or Slurm can be used to launch clusters or submit jobs, but within a Ray-run pipeline orchestration is handled within Ray.
+```
+
+---
+
+## Advanced Topics
+
+```{dropdown} What if I need a fan-in or reduce operation (not just map or scatter)?
+
+The current model supports scatter and map, but not reduce or fan-in across tasks (for global aggregations). For most data curation workflows, this is not a limitation; custom logic can be used where needed.
+```
+
+```{dropdown} How do I manage large, multi-node training jobs with fine-grained control (for example, PyTorch-NCCL setup)?
+
+For multi-node operations, you're responsible for managing communication setup (for example, establishing NCCL channels for distributed training within the relevant stage's setup logic).
+```
+
+```{dropdown} Are there examples of multi-type data (text, image, audio, video) pipelines?
+
+Yes. NeMo Curator supports multiple data modalities including {ref}`text <gs-text>`, {ref}`image <gs-image>`, {ref}`audio <gs-audio>`, and {ref}`video <gs-video>` (links to quickstart guides for each modality).
+```
+
+---
+
+## Support, Communication, and Community
+
+```{dropdown} Where can I get support, ask questions, or contribute?
+
+Support and discussion channels are available on Slack, Teams, and GitHub. If you build innovative filters or features for your locale, please engage and contribute back. Regular calls and community check-ins are offered.
```

Comment:
What's the Slack? And maybe Teams should be removed.

Reply:
it looks like our Discussions tab may be behind NVIDIA auth https://github.com/NVIDIA-NeMo/Curator/discussions

i'll try to find out what the slack is